visualization/dashboard/src/MLIDashboard/Interfaces/IExplanationDashboardProps.ts

```diff
@@ -37,6 +38,25 @@ export interface IExplanationDashboardProps {
     ) => Promise<any[]>;
 }
 
+export enum TelemetryLevels {
```

Comment:
My first reaction is that these should maybe go in their own file, but on more reflection it kinda depends on how you've laid out other stuff. Up to you.

Reply:
Yes, adding new files for each of the different interfaces may be clearer for future developers.